app/Model/Server.php

```diff
@@ -6935,6 +6947,95 @@ private function generateServerSettings()
                     'null' => true
                 ]
             ),
+            'SimpleBackgroundJobs' => [
+                'branch' => 1,
+                'enabled' => [
+                    'level' => 2,
+                    'description' => __('Enables or disables background jobs with Supervisor backend.'),
+                    'value' => false,
+                    'test' => 'testBool',
+                    'type' => 'boolean'
+                ],
+                'redis_host' => [
+                    'level' => 2,
+                    'description' => __('The host running the redis server to be used for background jobs.'),
+                    'value' => '127.0.0.1',
+                    'test' => 'testForEmpty',
+                    'type' => 'string'
+                ],
+                'redis_port' => [
+                    'level' => 2,
+                    'description' => __('The port used by the redis server to be used for background jobs.'),
+                    'value' => 6379,
+                    'test' => 'testForNumeric',
+                    'type' => 'numeric'
+                ],
+                'redis_database' => [
+                    'level' => 2,
+                    'description' => __('The database on the redis server to be used for background jobs. If you run more than one MISP instance, please make sure to use a different database on each instance.'),
```

Comment:
True, different namespaces could be used, though to avoid potential mess-ups (which would be on us of course), using a different db for each instance is a nice way to separate concerns. Also easier to just flush DBs when needed without affecting other instances.

Reply:
added some text to make it clear